packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -2179,7 +2179,8 @@ export function attach(
         if (
           elementType === ElementTypeFunction ||
           elementType === ElementTypeClass ||
-          elementType === ElementTypeContext
+          elementType === ElementTypeContext ||
+          elementType === ElementTypeMemo
```

Comment:
Oh nice fix. We should probably do the same for `ElementTypeForwardRef` too?

Reply:
updated